apps/lend/src/entities/chain/lib/chain-info.ts

```diff
@@ -2,7 +2,12 @@ import useStore from '@/store/useStore'
 import { useMemo } from 'react'
 import { FETCHING, PartialQueryResult, READY } from '@/shared/lib/queries'
 
-export const useChainId = (): PartialQueryResult<ChainId> => {
+export const useApi = (): PartialQueryResult<Api> => {
```

Comment:
Put jsdoc `@deprecated` tag or something like this. To don't use it and don'd forget to delete in a future.

Reply:
:thinking: why?

Reply:
I think it's a temporary hook and it shouldn't work that way. So I would label it somehow so I don't use it too often.